lib/t/push_notifications/dispatch_job.ex

```diff
@@ -41,6 +42,27 @@ defmodule T.PushNotifications.DispatchJob do
     end
   end
 
+  defp handle_type(type, args)
+       when type in ["timeslot_accepted_now"] do
+    %{"match_id" => match_id, "receiver_id" => receiver_id, "slot" => slot} = args
+
+    if match = alive_match(match_id) do
+      timeslot =
+        Matches.Timeslot |> where(match_id: ^match_id, selected_slot: ^slot) |> Repo.one()
+
+      if timeslot do
+        Matches.notify_timeslot_started(match)
```

Comment:
for "now slots" this can be done from `save_timeslots` function, there is a significant delay possible with oban jobs